### PR TITLE
Enable dynamic dhcpd ip addresses to enable IPI installs

### PR DESCRIPTION
**Pull request by walidshaari**
Enable Infrastructure provisioned installs/automated installs for OKD4/OCP4

solves issue #144

### CODE REVIEW

File: templates/dhcpd.conf.j2 [hunk 1]
```diff
@@ -28,8 +28,8 @@ max-lease-time 14400;
 		host {{ o.name }} { hardware ethernet {{ o.macaddr }}; fixed-address {{ o.ipaddr }}; }
 {% endfor %}
 {% endif %}
-		# this will not give out addresses to hosts not listed above
-		deny unknown-clients;
+		# uncomment below if you do not want to give out addresses to hosts not listed above
+		# deny unknown-clients;
```
Comment (christianh814) [inline templates/dhcpd.conf.j2:32]: I would just rather make this a variable than just comment it out.

**Comment by walidshaari** [comment]
yes, I was thinking of this too today, a variable do you want IPI? and this
will affect the DNS two entries for API and ingress too

**Comment by christianh814** [comment]
How will it affect the DNS portion?

**Comment by walidshaari** [comment]
IPI requires two distinct unused ips for keepalived, not the whole DNS records you are outputting right now

**Pull request closed without merging**